base_layer/common_types/src/tari_address/mod.rs

```diff
@@ -246,7 +246,7 @@ impl TariAddress {
 
     /// Construct Tari Address from hex
     pub fn from_base58(hex_str: &str) -> Result<TariAddress, TariAddressError> {
-        if hex_str.len() < 47 {
+        if hex_str.len() < 45 {
```

Comment:
This would be better with a descriptive const. Perhaps TARI_ADDRESS_BASE58_MIN_LENGTH

Reply:
yeah good idea